package.json

```diff
@@ -73,6 +73,12 @@
       ]
     }
   },
+  "dependencies": {
+    "@types/jest": "^29.5.5",
+    "@types/node": "^20.7.1",
+    "jest-environment-jsdom": "^29.7.0",
+    "typescript": "^5.2.2"
```

Comment:
These should be dev dependencies and locked

Reply:
Moved them to devDep